Code/Framework/AzToolsFramework/AzToolsFramework/AssetDatabase/AssetDatabaseConnection.cpp

```diff
@@ -1791,8 +1791,9 @@ namespace AzToolsFramework
             return true;
         }
 
-        bool AssetDatabaseConnection::PostOpenDatabase()
+        bool AssetDatabaseConnection::PostOpenDatabase(bool /*ignoreFutureAssetDBVersionError*/)
```

Comment:
nit: prefer `[[ maybe_unused ]]`

Reply:
I thought we used [[maybe_unused]] when it was unused in some configurations, like a variable that's used in Debug and unused in Profile, and commented out the parameter when it's entirely unused?

I'll switch it for now.

Reply:
If the parameter is complete unused then it should be just be deleted.

Reply:
It's a virtual function and it's used elsewhere